app/views/search/index.slim

```diff
@@ -33,4 +34,7 @@ article#search-results
 article#runs
   - if @results[:games].present?
     h4 Runs
-    = render partial: 'shared/run_table', locals: table_locals(:games, games: @results[:games])
+    .card.shadow = render partial: 'shared/run_table', locals: { \
+      runs: Run.by_game(@results[:games]).nonempty.unarchived, \
+      cols: %i[runner timer name uploaded] \
```

Comment:
`timer` should probably be `time`, unless you are explicitly trying to get the timer to show.  If it is the case, timer is not showing, and time would be a nice addition to this output so you know what kind of run you are about to look at.